tree/ntuple/v7/test/ProxiedSTLContainerLinkDef.h

```diff
@@ -20,7 +20,7 @@
 #pragma link C++ class std::map<int, std::vector<CustomStruct>>+;
 #pragma link C++ class std::map<std::string, float>+;
 #pragma link C++ class std::map<char, std::map<int, CustomStruct>>+;
-#pragma link C++ class std::map<float, std::map<char, std::int32_t>>+;
+#pragma link C++ class std::map<int, std::map<char, float>>+;
```

Comment:
Unrelated to this PR, we also have the (preexisting) test on:
```
#pragma link C++ class std::unordered_map<float, std::vector<bool>>+;
```
Using a float (or double) as a key is a "bad" items as float comparison is unstable.  I have seen case where depending on how the code was optimized, the insertion and retrieval from the map was actually using opcode with different precision leading to a corrupted map.